### ISSUE TITLE
Adjustments to sales query tool

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3102]
Based off the initial review, make these design adjustments to the sales query tool: 

- When I have the page at 90% or higher, *Sales Volume total not available..." message is real close to the horizontal scroll bar. Can we move it a few pixels lower?

- The leading in "Royalty Value Prior to Allowances" and "Royalty Value Less Allowances" is more than needed. Can we reduce it?

**Comment by Maroyafaied** [comment]
Erin and Jeff please discuss which of these have to be done before we publish sales data and which can wait till after.

**Comment by Maroyafaied** [comment]
Erin will review the items listed here and see if any of them also came out of user research findings , she will organize and open issues with appropriate labels.

**Comment by eelzi-ONRR** [comment]
@greensomes I removed the items related to add a column since that's covered by another issue from user research and not rated high priority.

Of the two items left in this issue: 

- The leading on the column labels is less important for launch

-  Can we fix the spacing of the N/A note at 90% and higher, before launch? I think that will make it be more visible to users.

**Comment by greensomes** [comment]
Hi @eelzi-ONRR! I've made some changes that are [ready for review](https://nrrd-preview.app.cloud.gov/sites/3102-adjustments-to-sales-query-tool/query-data). Let me know what you think. Thanks!

**Comment by greensomes** [comment]
@eelzi-ONRR, the spacing between the scrollbar and note text has been bumped. You can review it [here](https://nrrd-preview.app.cloud.gov/sites/3102-adjustments-to-sales-query-tool/query-data/).

**Comment by eelzi-ONRR** [comment]
@greensomes It looks great now! Thank you!
